tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist=py35,py36
+
+[testenv]
+deps=pytest
+commands =
```

Comment:
I would highly suggest adding `flake8` test here as well.

Reply:
I agree. I plan to integrate proper testing in the next pr.